webpack.config.js

```diff
@@ -14,41 +14,69 @@ const config = {
   },
 
   module: {
-    rules: [{
-      test: /\.(js|jsx)$/,
-      exclude: /node_modules/,
-      include: [
-        path.resolve(__dirname, 'src'),
-      ],
-      use: {
-        loader: 'babel-loader',
-        query: {
-          plugins: ['lodash'],
-        },
-      }
-    }],
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        include: [
+          path.resolve(__dirname, 'src'),
+        ],
+        use: {
+          loader: 'babel-loader',
+          query: {
+            plugins: [
+              'lodash', 
+              ["@nutanix-ui/babel-plugin-prism-import", {
+                "libraryName": "prism-reactjs",
+                "style": "less"
+              }]
+            ],
+          },
+        }
+      },
+      {
+        test: /\.less$/,
+        use: [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'less-loader',
+          }
+        ]
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader'
+        ]
+      },
+        // For loading fonts.
+        {
+          test: /\.(woff|woff2|eot|eot\?iefix|ttf|svg|gif)$/,
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]'
+          }
+        }
+    ]
   },
 
-  externals: {
-    react: {
-      root: 'React',
-      commonjs2: 'react',
-      commonjs: 'react',
-      amd: 'react',
-    },
-    'react-dom': {
-      root: 'ReactDOM',
-      commonjs2: 'react-dom',
-      commonjs: 'react-dom',
-      amd: 'react-dom'
-    },
-    'prop-types': {
-      root: 'PropTypes',
-      commonjs2: 'prop-types',
-      commonjs: 'prop-types',
-      amd: 'prop-types',
-    },
-  },
+  externals: [
+    'react',
+    'react-dom',
+    'core-js',
+    'd3-interpolate',
+    'd3-scale',
+    'd3-shape',
+    'lodash',
+    'prism-reactjs',
```

Comment:
I think you might have to do regex filtering for lodash and prism-reactjs because bable will tranlate
import { Button } from 'prism-reactjs';
to 
import Button from 'prism-reactjs/lib/v2/Button/Button.js';


'^prism-reactjs.*'
'^lodash.*'

Reply:
You can choose to address this in your next submit if you want.

Reply:
Ok. Let's revisit these.